doc/source/cice_index.rst

```diff
@@ -139,6 +139,7 @@ either Celsius or Kelvin units).
    "daymo", "number of days in one month", ""
    "daycal", "day number at end of month", ""
    "days_per_year", ":math:`\bullet` number of days in one year", "365"
+   "day_init", ":math:`\bullet` the initial day of the month", ""
```

Comment:
The model can start on any year, month, day, and sec now.  I agree that since we don't have history restart capability, this could be an issue.  But that has always been the case.  CESM starts and stops on various days at times and the monthly averages are useless.  What we really need are history restarts.

Reply:
Hopefully we have a note about that limitation in the documentation!